### PR TITLE
Mission order

### PR DESCRIPTION
**Pull request by itsAndL**

### CODE REVIEW

File: app/models/mission_order.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class MissionOrder < ApplicationRecord
+  after_create :create_user_request
+
+  belongs_to :site
+  has_one :user_request, as: :requestable, dependent: :destroy
+  has_rich_text :description
+
+  enum indemnity_type: {none_indemnity_types: 0, expense_report: 1, flat_rate: 2}
+  enum accommodation: {none_accommodations: 0, apartment: 1, hotel: 2, other_accommodations: 3}
```
Comment (mohamedrez) [inline app/models/mission_order.rb:9]: remove none

File: app/models/mission_order.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  enum mission_type: {none_mission_types: 0, project: 1, site: 2}
```
Comment (mohamedrez) [inline app/models/mission_order.rb:10]: remove none

File: app/models/mission_order.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  enum transport_means: {none_transport_means: 0, car: 1, bus: 2, train: 3, motorbike: 4, taxi: 5, other_transport_means: 6}
```
Comment (mohamedrez) [inline app/models/mission_order.rb:11]: remove none

Comment (mohamedrez) [inline app/models/mission_order.rb:11]: transport_means needs to be transport_type , the types are plane, train, bus

**Comment by itsAndL** [comment]
https://github.com/mohamedrez/grh/assets/105809250/e436920f-84fc-4810-b8e8-0a274c5740a2

**Pull request merged**